docs/reference.md

```diff
@@ -420,6 +421,11 @@ OpenWhisk has a few system limits, including how much memory an action uses and
 * A user can change the limit when creating the action.
 * A container cannot have more memory allocated than the limit.
 
+### Per action logs (MB) (Default: 10MB)
+* The log limit M is in the range from [0MB..10MB] and is set per action in MB.
+* A user can change the limit when creating the action.
+* If a container writes more logs than the limit, the activation will contain the logs until that limit and truncated afterwards. The last logline will contain warning.
```

Comment:
- Just confirming.  "truncated afterward" means that no additional log entries will be added to the log file ?
- What happens when the size is set to 0MB?